local_server/qwen3_tts_server/local_server.py

```
@@ -0,0 +1,450 @@
+                break
+        if config_path is not None:
+            with config_path.open("r", encoding="utf-8") as f:
+                cfg = json.load(f)
+            tts_custom = cfg.get("tts_custom", {}) or {}
+            repo_root = config_path.parent.parent
+    except Exception:
+        tts_custom = {}
+        repo_root = None
+
+    model_path = tts_custom.get("model_path") or "/home/amadeus/models/qwen3_tts"
+    host = tts_custom.get("host") or "0.0.0.0"
```

Ruff (0.15.2)

[error] Possible binding to all interfaces

(S104)